stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -157,9 +157,17 @@ internal class ChatDomainImpl internal constructor(
      */
     override val errorEvents: LiveData<Event<ChatError>> = _errorEvent
 
+    /** the event subscription, cancel using repo.stopListening */
+    private var eventSubscription: Disposable? = null
+
+    /** stores the mapping from cid to channelRepository */
+    private val activeChannelMapImpl: ConcurrentHashMap<String, ChannelControllerImpl> = ConcurrentHashMap()
```

Comment:
Oh, it doesn't mean good things for me... It seems that yesterday I was wrong saying "Flow doesn't give us any enhancement right now"